app/models/appointment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+class Appointment < ApplicationRecord
+  belongs_to :patient_profile
+  belongs_to :service
+  enum status: { NOT_CONFIRMED: 0, CONFIRMED: 1, CANCELLED: 2, COMPLETED: 3 }
```

Comment:
COMPLETED should be 2 and CANCELLED should be 3

Reply:
Fixed